src/composables/transactionItemList.ts

```diff
@@ -0,0 +1,128 @@
+import { computed } from '@vue/composition-api';
+import {
+  IActiveMultisigTx,
+  IDefaultComposableOptions,
+  ITokenList,
+  ITransaction,
+  ITx,
+} from '../types';
+import {
+  AETERNITY_CONTRACT_ID,
+  defaultTransactionSortingCallback,
+  getInnerTransaction,
+  TX_FUNCTIONS,
+} from '../popup/utils';
+import { useTransactionTx } from './transactionTx';
+import { usePendingMultisigTransaction } from './pendingMultisigTransaction';
+import { useTransactionAndTokenFilter } from './transactionAndTokenFilter';
+import { i18n } from '../store/plugins/languages';
+
+interface TransactionItemList extends IDefaultComposableOptions {
+  isMultisig?: boolean,
+  token: string,
+}
+
+type ITransactionElement = ITransaction | IActiveMultisigTx
```

Comment:
In my opinion this is a really bed pattern to mix two totally different objects together. Paweł had already issues with this approach when we were trying to combine IAccount and IMultisigAccount. Here the differences are even bigger.

Reply:
In this case, I don't see another solution. We must handle both types in transaction list.

Reply:
OK. Let's leave it as it is. Maybe we'll come up with something better in the future.